st2common/st2common/policies/meta/BUILD

```diff
@@ -0,0 +1,5 @@
+# These are loaded by st2common.bootstrap.policiesregistrar
+resources(
+    name="policies_meta",
+    sources=["*.yaml"],
+)
```

Comment:
Adding this BUILD file broke a test that was checking directory contents. So, I assisted the test to account for it.